sdks/python/apache_beam/runners/worker/statesampler_slow_test.py

```diff
@@ -0,0 +1,92 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Tests for state sampler."""
+from __future__ import absolute_import
+
+import logging
+import time
+import unittest
+
+from apache_beam.runners.worker import statesampler_slow as statesampler
+from apache_beam.utils.counters import CounterFactory
+from apache_beam.utils.counters import CounterName
+
+
+class StateSamplerSlowTest(unittest.TestCase):
```

Comment:
<!--thread_id:cc_161077336_t; commit:4f9a58f5527e336297aebb9e13d5c6401a9a7c4a; resolved:1-->
<!--section:context-quote-->
> **robertwb** wrote:
> It would be better to run the same test on both implementations rather than have a copy. (Possibly conditionally test those parts that aren't implemented of course.)

<!--section:body-->
Acknowledged.